tests/unit/views/notifications/project_crew_notification.feature

```diff
@@ -0,0 +1,83 @@
+      | Rincewind | editor,promoter | Havelock Vetinari made Twoflower an editor and promoter of Ankh-Morpork 2010 |
+      | Vetinari  | editor,promoter | Havelock Vetinari made Twoflower an editor and promoter of Ankh-Morpork 2010 |
+      | Twoflower | crew            | Havelock Vetinari added you to the crew of Ankh-Morpork 2010                 |
+      | Rincewind | crew            | Havelock Vetinari added Twoflower to the crew of Ankh-Morpork 2010           |
+      | Vetinari  | crew            | Havelock Vetinari added Twoflower to the crew of Ankh-Morpork 2010           |
+
+  Scenario Outline: Vetinari has invited Twoflower
+    Given Rincewind and Twoflower are project crew in the project Expo 2010
+    When Vetinari invites Twoflower to the project with a role <role>
+    Then <user> gets notified <notification_string> about invitation.
+
+    Examples:
+      | user      | role            | notification_string                                                                               |
+      | Twoflower | editor          | You have been invited to be an editor of Ankh-Morpork 2010 by Havelock Vetinari                   |
+      | Rincewind | editor          | Twoflower has been invited to be an editor of Ankh-Morpork 2010 by Havelock Vetinari              |
+      | Vetinari  | editor          | Twoflower has been invited to be an editor of Ankh-Morpork 2010 by Havelock Vetinari              |
+      | Twoflower | promoter        | You have been invited to be a promoter of Ankh-Morpork 2010 by Havelock Vetinari                  |
+      | Rincewind | promoter        | Twoflower has been invited to be a promoter of Ankh-Morpork 2010 by Havelock Vetinari             |
+      | Vetinari  | promoter        | Twoflower has been invited to be a promoter of Ankh-Morpork 2010 by Havelock Vetinari             |
+      | Twoflower | editor,promoter | You have been invited to be an editor and promoter of Ankh-Morpork 2010 by Havelock Vetinari      |
+      | Rincewind | editor,promoter | Twoflower has been invited to be an editor and promoter of Ankh-Morpork 2010 by Havelock Vetinari |
+      | Vetinari  | editor,promoter | Twoflower has been invited to be an editor and promoter of Ankh-Morpork 2010 by Havelock Vetinari |
+      | Twoflower | crew            | You have been invited to join the crew of Ankh-Morpork 2010 by Havelock Vetinari                  |
+      | Rincewind | crew            | Twoflower has been invited to join the crew of Ankh-Morpork 2010 by Havelock Vetinari             |
+      | Vetinari  | crew            | Twoflower has been invited to join the crew of Ankh-Morpork 2010 by Havelock Vetinari             |
+
+  Scenario Outline: Twoflower has accepted the invite
+    Given Rincewind and Twoflower are project crew in the project Expo 2010
+    When Twoflower accepts the invitation to be an editor of project Expo 2010 with a role <role>
```

Comment:
`Given` is repeating across these tests. It should be specified once at the top of the file (in the `Background` section?)